abigen/src/event.rs

```diff
@@ -231,7 +231,7 @@ mod tests {
             anonymous: false,
         };
 
-        let e = Event::from(&ethabi_event);
+        let e = Event::from(&(ethabi_event.name.clone(), ethabi_event));
```

Comment:
And then each of those becomes
```suggestion
        let e = Event::from((&ethabi_event.name, &ethabi_event));
```

So we get rid of the unneeded `clone` here